test/Microsoft.AspNet.Mvc.FunctionalTests/Compiler/Resources/MvcTagHelpersWebSite.Customer.Index.html

```diff
@@ -0,0 +1,42 @@
+﻿
```

Comment:
you're using the same controller / action pair twice here.  the action is "Index" though that's implicit.  so, you need to extend the convention slightly.